Java invoke-bridge build failed

### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

I can't build the Java invoke-bridge as below.



### Service configuration (serverless.yml) content

```yaml
N/A
```


### Command name and used flags

mvn package

### Command output

```shell
% pwd
/Users/yyamano/work/serverless/lib/plugins/aws/invoke-local/runtime-wrappers/java
% mvn package
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------< com.serverless:invoke-bridge >--------------------
[INFO] Building invoke-bridge 1.0.1
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.amazonaws:aws-lambda-java-log4j:jar:1.5.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.254 s
[INFO] Finished at: 2022-03-10T15:49:38+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project invoke-bridge: Could not resolve dependencies for project com.serverless:invoke-bridge:jar:1.0.1: Failure to find com.amazonaws:aws-lambda-java-log4j:jar:1.5.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```


### Environment information

```shell
v3.7.3 from github.

% node ~/work/serverless/bin/serverless.js -v
Framework Core: 3.7.3
Plugin: 6.1.5
SDK: 4.3.2
```

--- a/lib/plugins/aws/invoke-local/runtime-wrappers/java/pom.xml
+++ b/lib/plugins/aws/invoke-local/runtime-wrappers/java/pom.xml
@@ -21,11 +21,6 @@
     </dependency>
     <dependency>
       <groupId>com.amazonaws</groupId>
-      <artifactId>aws-lambda-java-log4j</artifactId>
-      <version>1.5.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.amazonaws</groupId>
       <artifactId>aws-lambda-java-events</artifactId>
       <version>3.1.1</version>
     </dependency>

